open-metadata-implementation/governance-servers/open-lineage-services/open-lineage-services-server/src/main/java/org/odpi/openmetadata/governanceservers/openlineage/admin/OpenLineageOperationalServices.java

```diff
@@ -99,8 +99,8 @@ public void initialize(OpenLineageConfig openLineageConfig, OMRSAuditLog auditLo
 
                 this.graphBuilder = new GraphBuilder();
                 MockGraphGenerator mockGraphGenerator = new MockGraphGenerator();
-                QueryHandler queryHandler = new QueryHandler();
-                this.instance = new OpenLineageServicesInstance(mockGraphGenerator, queryHandler, localServerName);
+                GraphServices graphServices = new GraphServices();
+                this.instance = new OpenLineageServicesInstance(mockGraphGenerator, graphServices, localServerName);
             } catch (RepositoryErrorException e) {
                 log.error("{} Could not open graph database", "GraphBuilder constructor");
```

Comment:
This error log formats 2 simple strings.
Details about the exceptions is more useful.